modules/bridge/src/main/ts/ephox/bridge/components/dialog/Checkbox.ts

```diff
@@ -1,26 +1,25 @@
-import { FieldSchema, StructureSchema, ValueType } from '@ephox/boulder';
+import { StructureSchema, ValueType } from '@ephox/boulder';
 import { Result } from '@ephox/katamari';
 
-export interface CheckboxSpec {
```

Comment:
This was actually defined incorrectly in the original implementation since it's implemented as a form component so the types/schema should match.